From 0f0678dfc2df8767b1b0e325d5f39140310218d4 Mon Sep 17 00:00:00 2001
From: Joyce Ooi <joyce.ooi@intel.com>
Date: Wed, 21 Oct 2020 16:03:01 +0800
Subject: [PATCH 140/151] HSD #1508459581: ARM: dts: adjust the clock frequency
 and falling time for i2c1

commit  ace04c059af781b0e14bf4a61cfe0208e2060155 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

This patch adjusts the clock frequency and falling time for i2c1 as the
A10 display port does not work at 100kHz.

Signed-off-by: Joyce Ooi <joyce.ooi@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index b3d99e434223..53906d4868fc 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -128,9 +128,9 @@
 	 * adjust the falling times to decrease the i2c frequency to 50Khz
 	 * because the LCD module does not work at the standard 100Khz
 	 */
-	clock-frequency = <100000>;
-	i2c-sda-falling-time-ns = <6000>;
-	i2c-scl-falling-time-ns = <6000>;
+	clock-frequency = <50000>;
+	i2c-sda-falling-time-ns = <60000>;
+	i2c-scl-falling-time-ns = <60000>;
 
 	lcd: lcd@28 {
 		compatible = "newhaven,nhd-0216k3z-nsw-bbw";
-- 
2.26.1

